CHANGES.rst

```diff
@@ -43,6 +43,8 @@ Other
 
 - Fix typo in ``outlierifuoutput`` schema for ``kernel_ysize`` [#191]
 
+- Add ``AmiLgFitModel`` class and schema [#199]
```

Comment:
Overall Looks great!

One minor change, 1.8.0 was just released this morning. Would you move this changelog entry to the new 1.8.1 section?

Reply:
Thanks! Just moved it.